bmaptools/BmapHelpers.py

```diff
@@ -95,70 +93,61 @@ def program_is_available(name):
 
     return False
 
+def get_mount_point(path):
```

Comment:
This function is marked as public (does not start with "_"), is this intentional? I did not spot the user outside this module.

Reply:
actually, this method is not actually used so this can be removed